cmd/axelard/cmd/root.go

```diff
@@ -41,9 +41,10 @@ import (
 	"github.com/axelarnetwork/axelar-core/cmd/axelard/cmd/utils"
 	"github.com/axelarnetwork/axelar-core/config"
 	"github.com/axelarnetwork/axelar-core/vald"
+	axelarnettypes "github.com/axelarnetwork/axelar-core/x/axelarnet/exported"
```

Comment:
```suggestion
	axelarnet "github.com/axelarnetwork/axelar-core/x/axelarnet/exported"
```
we don't put the `type` suffix for the exported package.